Steps/UploadCodeCoverage.yml

```diff
@@ -145,14 +152,20 @@ steps:
         cmd = f'{UPLOAD_CMD} -f {cov_file} -Z'
         if COV_FLAG:
           cmd += f' -F {COV_FLAG}'
-        process = subprocess.Popen(
-          cmd,
-          stdout=subprocess.PIPE,
-          stderr=subprocess.PIPE,
-          shell=True)
-        output, error = process.communicate()
-        print(f"##[debug]{output.decode('utf-8')}")
+        loop = 0
+        process = 0
+        while loop < 5 and process == 0:
```

Comment:
I think this would be a little cleaner with a `for ... in range()` loop where `5` is replaced with a named constant.